src/test/scala/com/github/dnvriend/serializer/avro4s/AvroTestSpec.scala

```diff
@@ -105,7 +105,7 @@ trait Decoder[IN, OUT] {
 }
 
 case class Base64(value: String)
-case class AvroCommand(base64: Base64, oldSchema: Schema, newSchema: Schema)
+case class AvroSchemaEvolution(base64: Base64, oldSchema: Schema, newSchema: Schema)
```

Comment:
I think that `AvroCommand` can be confused with the "Command" trait used in domain (and generally with Akka persistence). I propose this name.